src/main/java/com/google/devtools/build/lib/windows/jni/WindowsFileOperations.java

```diff
@@ -65,12 +65,22 @@ private WindowsFileOperations() {
   private static final int DELETE_PATH_DIRECTORY_NOT_EMPTY = 3;
   private static final int DELETE_PATH_ACCESS_DENIED = 4;
 
+  private static final int READ_SYMLINK_OR_JUNCTION_SUCCESS = 0;
+  // READ_SYMLINK_OR_JUNCTION_ERROR = 1;
```

Comment:
Why comment here? Is `READ_SYMLINK_OR_JUNCTION_ERROR` set or used anywhere?

Reply:
For documentation, and to indicate that the value exists. The variable is not used anywhere (handled by `switch.default`).

Removed.